src/ScorimmoLead.php

```diff
@@ -0,0 +1,340 @@
+    }
+
+    /**
+     * @return string
+     */
+    public function getPropertyAddressLine1()
+    {
+        return $this->property_address_line_1;
+    }
+
+    /**
+     * @param string $property_address_line_1
+     */
+    public function setPropertyAddressLine1($property_address_line_1)
+    {
+        $this->property_address_line_1 = $property_address_line_1;
+    }
+
+    /**
+     * @return string
+     */
+    public function getPropertyZipcode()
+    {
+        return $this->property_zipcode;
+    }
+
+    /**
+     * @param string $property_zipcode
+     */
+    public function setPropertyZipcode($property_zipcode)
+    {
+        $this->property_zipcode = $property_zipcode;
+    }
+
+    /**
+     * @return string
+     */
+    public function getPropertyCity()
+    {
+        return $this->property_city;
+    }
+
+    /**
+     * @param string $property_city
+     */
+    public function setPropertyCity($property_city)
+    {
+        $this->property_city = $property_city;
+    }
+
+    /**
+     * @return string
+     */
+    public function getEmail()
+    {
+        return $this->email;
+    }
+
+    /**
+     * @param string $email
+     */
+    public function setEmail($email)
+    {
+        $this->email = $email;
+    }
+
+    /**
+     * @return string
+     */
+    public function getPhone()
+    {
+        return $this->phone;
+    }
+
+    /**
+     * @param string $phone
+     */
+    public function setPhone($phone)
+    {
+        $this->phone = $phone;
+    }
+
+    /**
+     * @return string
+     */
+    public function getOrigin()
+    {
+        return $this->origin;
+    }
+
+    /**
+     * @param string $origin
+     */
+    public function setOrigin($origin)
+    {
+        $this->origin = $origin;
+    }
+
+    /**
+     * @return string
+     */
+    public function getInterest()
+    {
+        return $this->interest;
+    }
+
+    /**
+     * @param string $interest
+     */
+    public function setInterest($interest)
+    {
+        $this->interest = $interest;
+    }
+
+    /**
+     * @return int
+     */
+    public function getSellerId()
+    {
+        return $this->seller_id;
+    }
+
+    /**
+     * @param int $seller_id
+     */
+    public function setSellerId($seller_id)
+    {
+        $this->seller_id = $seller_id;
+    }
+
+    /**
+     * @return string
+     */
+    public function getStatus()
+    {
+        return $this->status;
+    }
+
+    /**
+     * @param string $status
+     */
+    public function setStatus($status)
+    {
+        $this->status = $status;
+    }
+
+    /**
+     * Mandatory field for Scorimmo
+     * @return bool
+     */
+    public function isValid()
+    {
+        if(
+            empty($this->getPropertyType()) ||
+            empty($this->getLastName()) ||
+            empty($this->getPhone()) ||
+            empty($this->getOrigin()) ||
+            empty($this->getInterest())
+        ) {
+            return false;
+        }
+        return true;
```

Comment:
Can be simplified:
```php
return !(
    empty($this->getPropertyType()) ||
    empty($this->getLastName()) ||
    empty($this->getPhone()) ||
    empty($this->getOrigin()) ||
    empty($this->getInterest())
);
```
But it may be less legible 😄